RadarSDK/RadarAPIHelper.m

```diff
@@ -67,11 +67,13 @@ - (void)requestWithMethod:(NSString *)method
                 [req setHTTPBody:[NSJSONSerialization dataWithJSONObject:params options:0 error:NULL]];
             }
 
-            NSURLSessionConfiguration *configuration = [NSURLSessionConfiguration defaultSessionConfiguration];
+            NSURLSessionConfiguration *configuration;
             if (extendedTimeout) {
+                configuration = [NSURLSessionConfiguration defaultSessionConfiguration];
                 configuration.timeoutIntervalForRequest = 25;
                 configuration.timeoutIntervalForResource = 25;
             } else {
+                configuration = [NSURLSessionConfiguration ephemeralSessionConfiguration];
```

Comment:
Is this done for the purposes of `verified`? If so, a brief comment explaining why would be helpful. (Not obvious to me why we alternate between default and ephemeral)

Reply:
Will do

Reply:
Added